coralproject/talk
# Workaround RTE Button focus bug on IOS

## What does this PR do?
- Restore functionality for IOS when clicking on a RTE button should always focus into the contentEditable even if focus was somewhere else before.
- Inspired by https://github.com/facebook/draft-js/issues/1075#issuecomment-369700275

## How do I test this PR?

- Make sure your focus is outside of the RTE, e.g. select text outside of the RTE.
- Click on any of the RTE buttons bold, italic, etc.
- Focus should go inside RTE as expected on IOS
